### PR TITLE
Add APIs exposed to 'app' to docs

### PR DESCRIPTION
**Pull request by CWSpear**
This PR should be paired with https://github.com/zeit/hyperterm/pull/248.

That PR just adds `winCount` and `createWindow`, but I added the other two that were already exposed to the docs `plugins` and `config`, doing my best effort to explain what those were.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1054,6 +1054,32 @@ <h3 id="hterm"><a href="#hterm">The underlying terminal</a></h3>
 
         <h3 id="additional-apis"><a href="#additional-apis">Additional APIs</a></h3>
 
+        <p>The Election <code>app</code> objects are extended with the following properties:</p>
```
Comment (mauro-oto) [inline index.html]: @CWSpear `Election` here should be `Electron` 😛

Comment (CWSpear) [inline index.html]: Good catch: fixed.

**Comment by CWSpear** [comment]
Making some changes to zeit/hyperterm#248 which may reflect here. Will re-open after updates (or create a new one).

**Comment by CWSpear** [comment]
Closed in favor of https://github.com/zeit/hyperterm-website/pull/25

**Pull request closed without merging**